common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -92,16 +94,20 @@
     </%doc>
     <%
         from django.template import Template, Context
-        return Template("""
-            {% load render_bundle from webpack_loader %}
-            {% render_bundle entry %}
-            <script type="text/javascript">
-                {% autoescape off %}{{ body }}{% endautoescape %}
-            </script>
-        """).render(Context({
-            'entry': entry,
-            'body': capture(caller.body)
-        }))
+        try:
+            return Template("""
+                {% load render_bundle from webpack_loader %}
+                {% render_bundle entry %}
+                <script type="text/javascript">
+                    {% autoescape off %}{{ body }}{% endautoescape %}
+                </script>
+            """).render(Context({
+                'entry': entry,
+                'body': capture(caller.body)
+            }))
+        except IOError as e:
+            # Don't break Mako template rendering if the bundle or webpack-stats can't be found, but log it
```

Comment:
1. This is a call to the Django template engine inside of a Mako def which is used in Mako templates.
2. Idk, yeah, there can always be other issues. Catching one is better than catching none, IMO.
3. If it happened for real the page would render whatever UI was rendered on the server and wouldn't include the JavaScript bundle on the page. IMO that's better than a 500 page, but others may disagree.